apps/web/src/App.tsx

```diff
@@ -1,27 +1,13 @@
-import React from "react";
-import { Link } from "ui";
-import "./App.css";
+import React from 'react'
```

Comment:
In TS, the convention is to import React as:

```suggestion
import * as React from 'react'
```